src/hotspot/share/gc/shared/stringdedup/stringDedupProcessor.cpp

```diff
@@ -25,6 +25,7 @@
 #include "precompiled.hpp"
 #include "classfile/javaClasses.inline.hpp"
 #include "classfile/stringTable.hpp"
+#include "gc/shared/collectedHeap.hpp"
```

Comment:
This include as well as the include for perfData.hpp (line 45) could be removed.